CHANGELOG.md

```diff
@@ -0,0 +1,102 @@
+# FinOps Open Cost and Usage Specification Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+**Added:**
+
+- `RegionId` column
+
+**Changed:**
+
+- `Column naming convention` attribute includes the following new provisions:
```

Comment:
Do we want to track new requirements in an existing spec as an addition or a change?